x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -78,7 +82,35 @@ function getTabs(context, logStackframes) {
   ];
 }
 
-function DetailView({ errorGroup, urlParams, location }) {
+interface ErrorGroupData {
+  error: Error;
+  occurrencesCount: number;
+  groupId: string;
+}
```

Comment:
Fyi: I have another PR open that creates this interface in the backend, which the frontend can consume:
https://github.com/elastic/kibana/pull/26801/files#diff-3b25ebd78cc5ddf47a53f268eb895f64R12